attractions.html

```diff
@@ -1,170 +1,260 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-
-<head>
+  <head>
     <!--    Meta Data -->
-    <meta charset="utf-8">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-
-    <title>Savannah Safari Attractions - Attractions & Restaurants</title>
-    <meta name="description" content="Savannah Safari Attractions">
-    <meta name="author" content="UoL Team Web Dev 41">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="Savannah Safari Attractions" />
+    <meta name="author" content="UoL Team Web Dev 41" />
+    <title>Savannah Safari Attractions - Attractions and restaurants</title>
 
     <!--    css styles to be loaded -->
-    <link rel="stylesheet" href="assets/css/siteWide/style.css?v=1234234234">
+    <link rel="stylesheet" href="assets/css/siteWide/style.css?v=1234234234" />
 
     <!--    css styles for mix it up to be loaded -->
-    <link href="assets/css/mixitUp/reset.css" rel="stylesheet" />
-    <link href="assets/css/mixitUp/style.css" rel="stylesheet" />
+    <!-- <link href="assets/css/mixitUp/reset.css" rel="stylesheet" /> -->
```

Comment:
I will comment these back shortly. It was just easier to compare pages and see inconsistencies. Either way it's not required for this week yet anyway.

Reply:
Ok